services/nvidia-gpu-operator/24.6.2/helmrelease/extra-images.txt

```diff
@@ -1,7 +1,7 @@
 nvcr.io/nvidia/k8s/container-toolkit:{{ regexReplaceAllLiteral "-.+$" .Values.toolkit.version "" }}-ubuntu20.04
 ghcr.io/mesosphere/dkp-container-images/nvcr.io/nvidia/k8s/container-toolkit:{{ regexReplaceAllLiteral "-.+$" .Values.toolkit.version "" }}-ubi8-d2iq.0
```

Comment:
I think that both these should be bumped to `v1.16.2` and go back to official release for `-ubi8-d2iq.0` version.